cloudferrylib/os/actions/check_config_quota_neutron.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import utils as utl
+
+
+LOG = utl.get_log(__name__)
+
+
+class CheckConfigQuotaNeutron(action.Action):
```

Comment:
Please add docstring, should include:
 - Overview of the task (what it does)
 - Task dependencies (does it depend on any other task)
 - Task dependents (do other tasks depend on this one)
 - Config options required used